src/js/tabs/history.js

```diff
@@ -73,10 +69,13 @@ HistoryTab.prototype.angular = function (module) {
       }
     };
 
+    // History states
+    $scope.$watch('loadState.transactions',function(){
+      $scope.historyState = !$scope.loadState.transactions ? 'loading' : 'ready';
+    });
 
-    $scope.advanced_feature_switch = Options.advanced_feature_switch;
-
-    $scope.orderedTypes = ['sent','received','gateways','trades','orders','other'];
+    //$scope.typeUsage = [];
```

Comment:
I didn't take out any code that was already there. I'm not sure if it'll be needed later. @vhpoet can I take out these commented out lines?